### ISSUE TITLE
Lost reference to Store.State

### ISSUE THREAD

**Issue by lukasportal** [open-wa/wa-automate-nodejs#2526]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### What type of session are you experiencing this issue on?

Tested and experienced on both

### What type of host account are you experiencing this issue on?

Tested and experienced on both

### Mode

EASY API/CLI

### Current Behavior

WhatsApp just released an update and now we no longer have a reference to Store.State
using the latest version
I went through all modules available trying to find anything related to state, but nothing useful found
This module simply doesn't exist anymore as far as I know

so `window.Store.State.Socket` isn't working anymore 

### Expected Behavior

`window.Store.State` should not be null or undefined

### Steps To Reproduce

Launch it (QR Code won't work with an error message saying could not read property 'Socket' of undefined ('Store.State.Socket')

### DEBUG INFO

```markdown
any
```

### Environment

```markdown
any
```

**Comment by MarcosDLR** [comment]
Solution:

Change this in /src/lib/wapi.js

if (typeof webpackJsonp === 'function') webpackJsonp([], {[parasite]: (x, y, z) => getStore(z)}, [parasite]); 
        // else webpackChunkbuild.push([[parasite], {}, function (o, e, t) {let modules = []; for (let idx in o.m) {modules.push(o(idx));}	getStore(modules);}]);        
        else webpackChunkwhatsapp_web_client.push([[parasite], {}, function (o, e, t) {let modules = []; for (let idx in o.m) {modules.push(o(idx));}	getStore(modules);}]);        
        
  And change to this version of ws in /src/config/pupperteer.config.ts
  
  export const useragent = createUserAgent('2.2204.13')

**Comment by lukasportal** [comment]
didnt work, Store.State is still undefined

**Comment by smashah** [comment]
was on a flight. one sec

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.31.7](https://github.com/open-wa/wa-automate-nodejs/compare/4.31.6...4.31.7) (2022-02-25)

- 🚑  fix #2526 [`#2526`](https://github.com/open-wa/wa-automate-nodejs/issues/2526)
- Release 4.31.7 [`4ea3762`](https://github.com/open-wa/wa-automate-nodejs/commit/4ea3762d155f8ca85b917b934d8366c29599aba9)
- 🥚 patches update [`f7ed2f8`](https://github.com/open-wa/wa-automate-nodejs/commit/f7ed2f88ea4af297a9def7b29722cec9ce3905b9)
- updated types-only package [`b4b1979`](https://github.com/open-wa/wa-automate-nodejs/commit/b4b1979d2112f3b3f1cd5699b2e534c1256ed1a3)
